ert_storage/graphql/ensembles.py

```diff
@@ -45,12 +45,14 @@ class Meta:
 
     class Arguments:
```

Comment:
Wondering where Arguments are used, is this for graphql need?

Reply:
It's how you tell graphene's `Mutation` class which arguments are possible to use.